src/components/AvatarCropModal/Slider.js

```diff
@@ -26,12 +26,13 @@ const defaultProps = {
 
 // This component can't be written using class since reanimated API uses hooks.
 const Slider = (props) => {
+    const sliderValue = props.sliderValue;
```

Comment:
How is this change related? Maybe I'm missing something here...

Reply:
I guess it's the same reason as explained in https://github.com/Expensify/App/pull/18073/files#r1187092253